desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTable/StorageBrowserTable.scss

```diff
@@ -0,0 +1,55 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+@use 'variables' as vars;
+@use 'mixins';
+
+.hue-storage-browser__table {
+  @include mixins.hue-svg-icon__d3-conflict;
+
+  margin: 0 vars.$fluidx-spacing-s;
+
+  svg {
+    color: vars.$fluidx-blue-700;
+    margin-right: 6px;
+  }
+
+  thead {
+    //TODO:Check why height isnt applied in safari browser
+    height: 40px;
+    background-color: vars.$fluidx-gray-040;
+  }
+
+  th.ant-table-cell {
+    text-align: left;
+  }
+
+  .hue-storage-browser__table-row {
+    height: 40px;
+    border-bottom: solid 1px vars.$fluidx-gray-300;
+
+    td.ant-table-cell {
+      white-space: nowrap;
+      overflow-x: hidden;
+      text-overflow: ellipsis;
+      //ellipsis does not work without display as block or specifying max-width. Display as block not possible in table
```

Comment:
Thanks for clarifying, does it have to be pixels or can it be %, so `100%`? Otherwise we have a hard limit on the width of the cells, but perhaps this is okay, Not familiar with the design.

Reply:
Percentages don't tend to work unless a parent has a specific width mentioned. Ill have a look into removing the hard limit